### PR TITLE
#166238090 Build Email Verification Service

### PR DESCRIPTION
**Pull request by chialuka**
#### What does this PR do?

- Add email verification service for new users when they sign up.

#### Description of Task to be completed?

- Add email confirmation column to database users table

- Send new users email when they register with link for verifying their email

- Add email verification endpoint 

- Write tests to cover feature added

#### How should this be manually tested?

- Clone the repo and run `npm run test` from the server folder

- From Postman, make a fresh sign up request. Open the email received on registration and click on the verify email link.

- Make a patch request using this URL on Postman and a message confirming the email has been verified will be received

#### Any background context you want to provide?
This feature enables the app administrators to verify the emails of users when they register. This will ensure that users only register with emails they have access to.

#### What are the relevant pivotal tracker stories?
[#166238090](https://www.pivotaltracker.com/story/show/166238095)

#### Screenshots (if appropriate)
Screenshot (107)

Screenshot (106)

### CODE REVIEW

File: server/config/seed.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const debug = Debug('dev');
+const { encryptPassword } = encrypt;
+
+/**
+ * Insert super admin after tables are created
+ * @name insertSuperAdmin
+ * @returns {String} details of insert
+ */
+const insertSeed = async () => {
+  const hashedPassword = encryptPassword('nonsoDrums');
```
Comment (tolulope-od) [inline server/config/seed.js:14]: Haha I see what you did here! @LukasChiama

**Review by tolulope-od — APPROVED** [review]
Great job here! @LukasChiama I really learned a lot just reading through this! Awesome!

File: server/config/seed.js (hunk 1 above)
Comment (chialuka) [reply server/config/seed.js:14]: Lol. Thanks a lot for the review. I've learned a whole lot from the team

**Review by ayodejiAA — APPROVED** [review]
Looks good. Well done, bro.

File: server/config/seed.js (hunk 1 above)
Comment (chiboycalix) [reply server/config/seed.js:14]: Hahahaha. hashing `nonsoDrums` nice one. I love this team

File: server/utilities/emailModule.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import sendGridMail from '@sendgrid/mail';
```
[the remark this answers is not in the document]
Comment (chiboycalix) [reply server/utilities/emailModule.js:1]: Good job. Your code is easy to understand.

**Review by chiboycalix — APPROVED** [review]

**Review by Nerocodes — APPROVED** [review]

**Pull request merged**